assets/sidecar/backstage-cr-patches.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: rhdh.redhat.com/v1alpha3
+kind: Backstage
+metadata:
+  name: ai-rh-developer-hub
+  namespace: ai-rhdh
+spec:
+  application:
+    extraFiles:
+      secrets:
+        - name: rhdh-rhoai-bridge-token
+          mountPath: /opt/app-root/src
+          key: token
+  deployment:
+    patch:
+      spec:
+        template:
+          spec:
+            containers:
+              - name: backstage-backend
+              - env:
+                  - name: POD_IP
+                    valueFrom:
+                      fieldRef:
+                        fieldPath: status.podIP
+                  - name: POD_NAMESPACE
+                    valueFrom:
+                      fieldRef:
+                        fieldPath: metadata.namespace
+                envFrom:
+                  - secretRef:
+                      name: rhdh-rhoai-bridge-token
+                image: quay.io/gabemontero/model-catalog-bridge-location:latest
+                imagePullPolicy: Always
+                name: location
+                ports:
+                  - containerPort: 9090
+                    name: location
+                    protocol: TCP
+                volumeMounts:
+                  - mountPath: /opt/app-root/src/dynamic-plugins-root
+                    name: dynamic-plugins-root
+                workingDir: /opt/app-root/src
+              - env:
+                  - name: STORAGE_TYPE
+                    value: ConfigMap
+                  - name: RHDH_TOKEN
+                    # or pull value from $RHDH_TOKEN env var locally
+                    value: foo
+                  - name: POD_IP
+                    valueFrom:
+                      fieldRef:
+                        fieldPath: status.podIP
+                  - name: POD_NAMESPACE
+                    valueFrom:
+                      fieldRef:
+                        fieldPath: metadata.namespace
+                envFrom:
+                  - secretRef:
+                      name: rhdh-rhoai-bridge-token
+                image: quay.io/gabemontero/model-catalog-bridge-storage-rest:latest
```

Comment:
will switch to our redhat-ai-dev image after review from you all and some more testing wrt trying to get the app-config set up for allowed hosts and external tokens